File: Sources/echoscrape/Utilities/logger.swift
import Rainbow

public class Logger {
    static func success(_ message: String) {
        print("\("✖".red) \(message)")
    }
    static func fail(_ message: String) {
        print("\("✔".green) \(message)")
    }
    static func info(_ message: String) {
        print("\("ℹ".blue) \(message)")
    }
}

File: lib/logger/logger.go
package logger

import (
	"fmt"

	"github.com/fatih/color"
)

func Success(message string) {
	green := color.New(color.FgGreen)
	fmt.Printf("%s %s\n", green.Sprint("✔"), message)
}

func Fail(message string) {
	red := color.New(color.FgRed)
	fmt.Printf("%s %s\n", red.Sprint("✖"), message)
}

func Info(message string) {
	blue := color.New(color.FgBlue)
	fmt.Printf("%s %s\n", blue.Sprint("ℹ"), message)
}
